tests/queries/0_stateless/02668_parse_datetime_in_joda_syntax.sql

```diff
@@ -321,7 +321,15 @@ select parseDateTime64InJodaSyntaxOrZero('2024-10-09 10:30:10.123', 'yyyy-dd-MM
 select parseDateTime64InJodaSyntaxOrZero('wrong value', 'yyyy-dd-MM HH:mm:ss.SSS') = toDateTime64('1970-01-01 00:00:00.000', 3);
 select parseDateTime64InJodaSyntaxOrZero('2023-02-29 11:22:33America/Los_Angeles', 'yyyy-MM-dd HH:mm:ssz') = toDateTime64('1970-01-01 00:00:00', 0);
 select parseDateTime64InJodaSyntaxOrZero('', '') = toDateTime64('1970-01-01 00:00:00', 0);
-select parseDateTime64InJodaSyntaxOrZero('2177-10-09 10:30:10.123', 'yyyy-MM-dd HH:mm:ss.SSS') = toDateTime64('1970-01-01 00:00:00.000', 3);
+select parseDateTime64InJodaSyntaxOrZero('2177-10-09 10:30:10.123', 'yyyy-MM-dd HH:mm:ss.SSS') = toDateTime64('2177-10-09 10:30:10.123', 3);
+-- Random date time tests from 1900 to 2299
+select parseDateTime64InJodaSyntaxOrZero('1923-05-15 14:45:30.123', 'yyyy-MM-dd HH:mm:ss.SSS') = toDateTime64('1923-05-15 14:45:30.123', 3);
+select parseDateTime64InJodaSyntaxOrZero('2150-11-23 08:22:10.456', 'yyyy-MM-dd HH:mm:ss.SSS') = toDateTime64('2150-11-23 08:22:10.456', 3);
```

Comment:
It would be nice if you could add tests equivalent to l. 325ff to 02668_parse_datetime.sql